WooCommerce/WooCommerceUITests/Screens/Login/PrologueScreen.swift

```diff
@@ -1,20 +1,27 @@
-import Foundation
```

Comment:
Importing `Foundation` is unnecessary because there is no direct call to a Foundation specific type, such as `URL`.

I wonder why the Xcode Swift new file template still has `import Foundation` in it.

Besides, I'm pretty sure that XCTest brings Foundation with it.